docs/development/testguide.rst

```diff
@@ -1063,3 +997,93 @@ are in your local git repository::
 You can then run the tests with::
 
     root@5e2b89d7b07c:/astropy_src# python setup.py test
+
+.. _pytest-plugins:
+
+Pytest Plugins
```

Comment:
could you somehow mention that these are pytest-astropy plugins? Or something else, this just reads too generic, e.g. we also use other plugins not maintained by us, such as xdist, and pep8, etc and they are not detailed here.